Reply/app/src/main/res/layout/fragment_email.xml

```diff
@@ -16,128 +16,205 @@
-                    app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintTop_toBottomOf="@id/body_text_view"
-                    app:layout_constraintEnd_toEndOf="parent"
-                    android:overScrollMode="never" />
-
-            </androidx.constraintlayout.widget.ConstraintLayout>
-
-        </com.google.android.material.card.MaterialCardView>
-
-    </androidx.core.widget.NestedScrollView>
-
+            android:layout_marginTop="@dimen/grid_0_5"
+            android:layout_marginStart="@dimen/grid_1"
+            android:text="@{email.timeSent}"
+            android:textAppearance="?attr/textAppearanceBody2"
+            app:layout_constraintStart_toEndOf="@+id/sender_profile_image_view"
+            app:layout_constraintTop_toBottomOf="@+id/email_sender_text_view"
+            tools:text="2 hr ago" />
+        <TextView
+            android:id="@+id/email_sender_text_view"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/grid_1"
+            android:text="@{email.sender.fullName}"
+            android:textAppearance="?attr/textAppearanceBody2"
+            android:textColor="@android:color/black"
+            app:layout_constraintStart_toEndOf="@+id/sender_profile_image_view"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:text="Ali Connors" />
+        <TextView
+            android:id="@+id/email_recipients_text_view"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/grid_1"
+            android:textAppearance="?attr/textAppearanceBody2"
+            android:text="@{email.recipientsPreview}"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/sender_profile_image_view"
+            tools:text="To me, Ziad and Lily" />
+        <!-- TODO: Temporary hardcoded black, remove when updating to M3 Typography  -->
+        <TextView
+            android:id="@+id/body_text_view"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/grid_4"
+            android:scrollbars="vertical"
+            android:text="@{email.body}"
+            android:textAppearance="?attr/textAppearanceBody1"
+            android:textColor="@android:color/black"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/email_recipients_text_view" />
+        <androidx.constraintlayout.widget.Guideline
+            android:id="@+id/guidelineLeft"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            app:layout_constraintGuide_percent="0.5" />
+        <Button
```

Comment:
It looks like both of these buttons in the mocks use `inverse on surface` for their background, `on surface variant` for their text color, and no elevation. Do you want to create a button style, set the `materialButtonStyle` theme attribute to your new style in the theme, and then use that attr for these buttons?

Reply:
Yes, that seems like a good idea. However, I'll do this in a different ticket / PR.